kci_rootfs

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2019 Collabora Limited
+# Author: Michal Galka <michal.galka@collabora.com>
+#
+# This module is free software; you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the Free
+# Software Foundation; either version 2.1 of the License, or (at your option)
+# any later version.
+#
+# This library is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this library; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+
+import argparse
+import sys
+
+from kernelci.cli import Args, Command
+import kernelci.config.rootfs
+
+# -----------------------------------------------------------------------------
+# Commands
+#
+
+
+class cmd_validate(Command):
+    help = "Lists all rootfs configs"
```

Comment:
This should be rather something like "Validate the rootfs YAML configuration"

Reply:
fixed